packages/contractkit/src/wallets/azure-hsm-wallet.test.ts

```diff
@@ -298,6 +298,18 @@ describe('AzureHSMWallet class', () => {
                 normalizeAddressWith0x(knownAddress)
               )
             })
+
+            // https://github.com/ethereum/go-ethereum/blob/38aab0aa831594f31d02c9f02bfacc0bef48405d/rlp/decode.go#L664
+            test('signature with 0x00 prefix is canonicalized', async () => {
+              // nonce 65 with the mock private keys will produce an S value with the first byte as 0x00
```

Comment:
Maybe you could extract, and coy here the transaction. I know it will duplicated code, but if in the future, if someone change the example, because it needs to add something, could lead it to a nightmare to replicate the double 0, haha.
In this particular example, I prefer to know that the Tx belongs only to this tests, for it's specific requirements

Reply:
👍